gamemodes/terrortown/gamemode/client/cl_player_ext.lua

```diff
@@ -166,6 +166,54 @@ local function TargetPlayer()
 end
 net.Receive("TTT2TargetPlayer", TargetPlayer)
 
+local function UpdateCredits()
+	local ply = LocalPlayer()
+	if not IsValid(ply) then return end
+
+	ply.equipment_credits = net.ReadUInt(8)
+end
+net.Receive("TTT_Credits", UpdateCredits)
+
+local function UpdateEquipment()
+	local ply = LocalPlayer()
+	if not IsValid(ply) then return end
```

Comment:
we tend to name the local player `client`. Also I'm wondering if there is any scenario where `LocalPlayer()` returns a nil value. Especially if you make sure that the netmessage is sent from `TTT2PlayerReady` hook or so

Reply:
Actually I dont know. In this case I only copied the statement. I can remove it, if you can confirm I dont need it ^^